src/Simplex/Chat/Store/Groups.hs

```diff
@@ -518,6 +522,14 @@ getGroupMemberById db user@User {userId} groupMemberId =
       (groupMemberQuery <> " WHERE m.group_member_id = ? AND m.user_id = ?")
       (groupMemberId, userId)
 
+getGroupMemberByMemberId :: DB.Connection -> User -> GroupId -> MemberId -> ExceptT StoreError IO GroupMember
```

Comment:
why this method is even needed - how did it work for reactions?

Reply:
moderation refers to a different member, not the one that sent the message